File: Objective-C/DYDotsViewDemo/ViewController.m
//
//  ViewController.m
//  DYDotsViewDemo
//
//  Created by Dwarven on 15/1/10.
//  Copyright (c) 2015年 Dwarven. All rights reserved.
//

#import "ViewController.h"
#import "DYDotsView.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet DYDotsView *dotsView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    DYDotsView * dots = [[DYDotsView alloc] initWithFrame:CGRectMake(0, 0, 200, 20)];
    [dots setNumberOfDots:@5];
//    [dots setDuration:@0.4];
    [dots setDotsColor:[UIColor redColor]];
    [dots startAnimating];
    [dots setCenter:self.view.center];
    [self.view addSubview:dots];
    
    [_dotsView startAnimating];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: Swift/DYDotsViewDemo-Swift/ViewController.swift
//
//  ViewController.swift
//  DYDotsViewDemo-Swift
//
//  Created by Dwarven on 15/1/11.
//  Copyright (c) 2015 Dwarven. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let dots = DYDotsView(frame: CGRect(x: 0, y: 0, width: 300, height: 30))
        dots.numberOfDots = 5
//        dots.duration = 0.4
        dots.dotsColor = .red
        dots.startAnimating()
        dots.center = self.view.center
        self.view.addSubview(dots)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

